servicecontrol/transports.md

```diff
@@ -74,6 +75,14 @@ In addition to the [connection string options of the transport](/transports/sql/
   * *Optional* `Subscriptions Table=<subscription_table_name>@<schema>` - to specify the schema.
   * *Optional* `Subscriptions Table=<subscription_table_name>@<schema>@<catalog>` - to specify the schema and catalog.
 
+## PostgreSQL
+
+In addition to the [connection string options of the transport](/transports/postgresql/connection-settings.md#connection-configuration) the following ServiceControl specific options are available in versions 5.10 and above:
+
+* `Queue Schema=<schema_name>` - Specifies a custom schema for the ServiceControl input queue.
+* `Subscriptions Table=<subscription_table_name>` - Specifies PostgreSQL subscription table name.
+  * *Optional* `Subscriptions Table=<subscription_table_name>@<schema>` - to specify the schema.
```

Comment:
This should perhaps be change to the below to match what's in the manifest file?
```suggestion
  * *Optional* `Subscriptions Table=schema.tablename` - to specify the schema.
```